Samples/BlazorSampleApp/ILGPUWebHost/ComputeSession.cs

```diff
@@ -0,0 +1,218 @@
+﻿// ---------------------------------------------------------------------------------------
+//                                    ILGPU Samples
+//                           Copyright (c) 2022 ILGPU Project
+//                                    www.ilgpu.net
+//
+// File: ComputeSession.cs
+//
+// This file is part of ILGPU and is distributed under the University of Illinois Open
+// Source License. See LICENSE.txt for details.
+// ---------------------------------------------------------------------------------------
+
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using ILGPU;
+using ILGPU.Runtime;
+using BlazorSampleApp.MandelbrotExplorer;
+
+
+
+namespace BlazorSampleApp.ILGPUWebHost
+{
+    /// <summary>
+    /// An individual compute session represents all compute resources for a single accelerator stream.    
+    /// 
+    /// This object must implement IDisposable as it allocates dedicated non-movable GPU accessible buffers.
+    /// </summary>
+#nullable disable
+    public class ComputeSession : IDisposable
+    {
+
+        // one or more kernels can be hosted in each compute session depending on the complexity of our calculation
+        private System.Action<AcceleratorStream, Index1D, ArrayView1D<int, Stride1D.Dense>, ArrayView1D<float, Stride1D.Dense>, int, ArrayView<int>> mandelbrot_kernel;
+
+
+        // SessionID allows individual Blazor sessions to "find" their compute session 
+        
+        public string SessionID { get; set; }
+
+        private bool _active = true;
+        public bool IsActive { get { return _active; } set { _active = value; } }
```

Comment:
nit: maybe we could simply this code to improve readability?

Reply:
done